tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -535,7 +535,8 @@
       "name": "envoy.zipkin",
       "config": {
         "collector_cluster": "zipkin",
-        "collector_endpoint": "/api/v1/spans",
+        "collector_endpoint": "/api/v2/spans",
```

Comment:
not sure if this causes any backward compatible issues? If so, I can close this PR